src/components/AddItems.jsx

```diff
@@ -35,6 +54,13 @@ export function AddItems() {
 				alert('Please select an option for date');
 				return;
 			}
+			// if (data) {
+			// 	if (currentItems.includes(normalizedItemName)) {
+			// 		alert('This item already exists in the list');
+			// 		event.target.reset();
+			// 		return;
+			// 	}
+			// }
```

Comment:
Was this just for troubleshooting? 

I think we should probably remove this before merging, but open to other opinions depending on the purpose for it being here.

Reply:
removed